vncviewer/parameters.cxx

```diff
@@ -101,10 +101,14 @@ IntParameter qualityLevel("QualityLevel",
                           8);
 
 BoolParameter maximize("Maximize", "Maximize viewer window", false);
-BoolParameter fullScreen("FullScreen", "Full screen mode", false);
+BoolParameter fullScreen("FullScreen", "Enable full screen", false);
+StringParameter fullScreenMode("FullScreenMode", "Full screen mode", "Current");
 BoolParameter fullScreenAllMonitors("FullScreenAllMonitors",
                                     "Enable full screen over all monitors",
-                                    true);
+                                    false);
```

Comment:
Should probably mention that this is deprecated. Or even hide it from `--help`.

Reply:
Hiding it from help might be good but requires structural changes to how parameters are listed. At the moment it might be easiest to annotate it as deprecated.

Reply:
I think `removeParam()` should do what you want.